qiskit_nature/circuit/library/ansatzes/ucc.py

```diff
@@ -171,6 +172,9 @@ def __init__(
                 setting does _not_ influence the `excitations`. When relying on the default
                 generation method (i.e. not providing a `Callable` to `excitations`), these will
                 always be constructed with respect to a `HartreeFock` reference state.
+            include_imaginary: boolean flag which when set to 'True' expands the ansatz to include
+                imaginary parts using twice the number of free parameters. Additional operators
+                are appended by the '_build_fermionic_excitation_ops()'
```

Comment:
```suggestion
            include_imaginary: boolean flag which when set to 'True' expands the ansatz to include
                imaginary parts using twice the number of free parameters.
```
This part is not relevant since this is non-public API.

I am also wondering whether we could call this, `include_complex_conjugate` which is a better description, imo